schemas/representations.md

```diff
@@ -308,3 +319,64 @@ type Bls12_381Signature bytes
 At the block level, this presents as a byte array, where the first byte is the discriminator (`0x00` or `0x01`) and the remainder is sliced to form either of the two types depending on the discriminator.
 
 `byteprefix` is not valid for unions where any of the constitutive types are _not_ `Bytes`.
+
+### enum string representation
+
+By default, an enum is simply represented as a string in the data model. An
+`enum` in the schema simply defines the list of possible strings that could be
+used in that position.
+
+**Example**
+
+```ipldsch
+type Status enum {
+	| NOPE
```

Comment:
Can we switch these out to TitleCase rather than SCREAMCASE, same as your latest examples in https://github.com/ipld/specs/pull/196#issuecomment-542009802?